src/OrchardCore/OrchardCore.ContentManagement.Abstractions/Handlers/ValidateContentContext.cs

```diff
@@ -7,13 +10,31 @@ public ValidateContentContext(ContentItem contentItem) : base(contentItem)
         }
 
         public ContentValidateResult ContentValidateResult { get; } = new ContentValidateResult();
+
+        public string BuildPrefix<TPart>(string memberName)
```

Comment:
See PR #6175 nameof doesn't work always - I have created ModelState extension method there, Probably we could use that logic here

Reply:
In the context of ContentParts I think that using nameof is enough. The only issue  would be with named parts, which is solved by setting the Prefix in validate context when running for each part.